specs/ui-popover/test-components.ts

```diff
@@ -0,0 +1,114 @@
+import { CommonModule } from '@angular/common';
+import { Component, NgModule } from '@angular/core';
+
+import { TsPopoverModule, TsPopoverPosition } from '@terminus/ui-popover';
```

Comment:
Formatting. 

also, fyi there is a webstorm formatting setting that will pop them on their own lines as they are added